webpack/assets/javascripts/react_app/components/common/SearchInput/SearchInput.test.js

```diff
@@ -0,0 +1,28 @@
+import toJson from 'enzyme-to-json';
+import { mount, shallow } from 'enzyme';
+import React from 'react';
+import SearchInput from './';
+
+jest.unmock('./');
+
+describe('Search Input', () => {
+  it('should render', () => {
+    const wrapper = shallow(<SearchInput searchValue='val' timeout={300} />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  it('should render with href', () => {
```

Comment:
Could you use some more descriptive name for the test? Since there's no href in the search field I guess this string is a copy-paste leftover.